2264/3/helpers/comment_rating_counter.rb

```diff
@@ -0,0 +1,50 @@
+require 'net/https'
+require 'uri'
+require 'json'
+
+# Rating Counter
+class CommentRatingCounter
+  URL = 'https://westcentralus.api.cognitive.microsoft.com'.freeze
+  PATH = '/text/analytics/v2.0/sentiment'.freeze
+  KEY = 'de8a560ccae541e08ec1a30dcdd191a4'.freeze
+
+  attr_reader :uri, :documents, :request
+
+  def initialize(texts)
+    @uri = URI(URL + PATH)
+    build_documents(texts)
```

Comment:
I should not probably do anything in initializer except for saving input 
def initialize(texts)
  @texts = texts
end